### ISSUE TITLE
Using @timestamp in the parameters list for email triggers error

### ISSUE THREAD

**Issue by iamrudra** [Yelp/elastalert#186]
Tried to follow [the documentation] (http://elastalert.readthedocs.org/en/latest/ruletypes.html#alerts) to create an alert and put in custom subject line. This is what I see on the terminal. 

    Traceback (most recent call last):
      File "/usr/lib/python2.7/runpy.py", line 162, in _run_module_as_main
        "__main__", fname, loader, pkg_name)
      File "/usr/lib/python2.7/runpy.py", line 72, in _run_code
        exec code in run_globals
      File "/root/elastalert/elastalert/elastalert.py", line 1208, in <module>
        client = ElastAlerter(sys.argv[1:])
      File "/root/elastalert/elastalert/elastalert.py", line 75, in __init__
        self.conf = load_rules(self.args)
      File "elastalert/config.py", line 296, in load_rules
        raise EAException('Error loading file %s: %s' % (rule_file, e))
    elastalert.util.EAException: Error loading file example_rules/example_blacklist.yaml: Could not parse file example_rules/example_blacklist.yaml: while scanning for the next token
    found character '@' that cannot start any token
      in "example_rules/example_blacklist.yaml", line 68, column 3

I tried putting in "@timestamp". It doesn't throw an error but then it shows "\<MISSING VALUE\>" text in the subject field. Any insights on this?

**Comment by Qmando** [comment]
This is a YAML parse error. You can't use @ at the start of a YAML token without quotes.
```
>>> import yaml
>>> yaml.load('test: @timestamp')
Traceback (most recent call last):
   ....
yaml.scanner.ScannerError: while scanning for the next token
found character '@' that cannot start any token
>>> yaml.load('test: "@timestamp"')
{'test': '@timestamp'}
```

**Comment by iamrudra** [comment]
I did dig a little into that. Give my below zero knowledge on YAML, I found [this SO answer](http://tex.stackexchange.com/questions/148994/problem-found-character-that-cannot-start-any-token) quite helpful. I was wondering if the document can be updated with the correct the syntax?

Also, The \<MISSING VALUE\> is because it cannot find the @timestamp field?

**Comment by Qmando** [comment]
I will update the documentation.

And yes, that is correct. Though it seems strange that ``@timestamp`` would be missing, unless you have also set ``timestamp_field``, in which case the subject arg would need to change as well.

**Comment by iamrudra** [comment]
Cool. Thanks!

**Comment by Qmando** [comment]
Fixed by #188
